manifests/server.pp

```diff
@@ -78,5 +78,8 @@
     Anchor['mysql::server::end']
   }
 
-
+Class['mysql::server'] -> Mysql_database <||>
+Class['mysql::server'] -> Mysql_user <||>
```

Comment:
It is not possible here due to the aim of patch: we want to use this define without defined class mysql::server(for remote server where mysql server is already installed)
I would offer to clean it up or at least leave as is